packages/core/useBroadcastChannel/index.ts

```diff
@@ -49,17 +50,21 @@ export function useBroadcastChannel<D, P>(options: UseBroadcastChannelOptions):
       error.value = null
       channel.value = new BroadcastChannel(name)
 
-      channel.value.addEventListener('message', (e: MessageEvent) => {
+      const listenerOptions = {
+        passive: true,
+      }
+
+      useEventListener(channel, 'message', (e: MessageEvent) => {
         data.value = e.data
-      }, { passive: true })
+      }, listenerOptions)
 
-      channel.value.addEventListener('messageerror', (e: MessageEvent) => {
+      useEventListener(channel, 'messageerror', (e: MessageEvent) => {
         error.value = e
-      }, { passive: true })
+      }, listenerOptions)
 
-      channel.value.addEventListener('close', () => {
+      useEventListener(channel, 'close', () => {
         isClosed.value = true
-      })
+      }, listenerOptions)
```

Comment:
Unsure if these parameters have any effect.

<img width="690" alt="截圖 2025-01-10 晚上9 39 06" src="https://github.com/user-attachments/assets/a751063c-04bf-419c-bba4-83706b7779e4" />

```suggestion
      const listenerOptions = {
        passive: true,
      }

      useEventListener(channel, 'message', (e: MessageEvent) => {
        data.value = e.data
      }, listenerOptions)

      useEventListener(channel, 'messageerror', (e: MessageEvent) => {
        error.value = e
      }, listenerOptions)

      useEventListener(channel, 'close', () => {
        isClosed.value = true
      }, listenerOptions)
```

Reply:
@ilyaliao Yes, for the same reason listed above (BroadcastChannel is part of the standard DOM API and the listened events are also standard).

I purposefully skipped custom events in my other PR yesterday (#4477). An example:

https://github.com/vueuse/vueuse/pull/4477/files#diff-015d17be17c2a02d87f6b40bc47f526a5e82a01ee334f2883d271c6d98543d21L193